includes/data-stores/class-wc-admin-reports-customers-data-store.php

```diff
@@ -216,17 +216,27 @@ protected function get_sql_query_params( $query_args ) {
 		$having_clauses = array();
 
 		$exact_match_params = array(
+			'name',
 			'username',
 			'email',
 			'country',
 		);
 
 		foreach ( $exact_match_params as $exact_match_param ) {
-			if ( ! empty( $query_args[ $exact_match_param ] ) ) {
-				$where_clauses[] = $wpdb->prepare(
-					"{$customer_lookup_table}.{$exact_match_param} = %s",
-					$query_args[ $exact_match_param ]
-				); // WPCS: unprepared SQL ok.
+			if ( ! empty( $query_args[ $exact_match_param . '_includes' ] ) ) {
+				$exact_match_arguments = esc_sql( $query_args[ $exact_match_param . '_includes' ] );
+				// 'country_includes' is a list of country codes, the others will be a list of customer ids.
+				$table_column          = 'country' === $exact_match_param ? $exact_match_param : 'customer_id';
+				$included              = implode( "','", explode( ',', $exact_match_arguments ) );
+				$where_clauses[]       = "{$customer_lookup_table}.{$table_column} IN ('{$included}')";
+			}
+
+			if ( ! empty( $query_args[ $exact_match_param . '_excludes' ] ) ) {
+				$exact_match_arguments = esc_sql( $query_args[ $exact_match_param . '_excludes' ] );
```

Comment:
`esc_sql()` needs to get called for each item in this array as we're building the `WHERE` clause.